thetis/configuration.py

```diff
@@ -195,7 +195,38 @@ def info(self):
         return result + super(PairedEnum, self).info()
 
 
-class FrozenHasTraits(HasTraits):
+class OptionsBase(object):
+    """Abstract base class for all options classes"""
+
+    def name(self):
```

Comment:
Yeah, since `HasTraits` is a metaclass, you need to interpose something else (ugh).  Like this perhaps?

```python

class OptionsBase(object):
    __metaclass__ = ABCMeta

class Interposer(OptionsBase, HasTraits):
    pass

class FrozenHasTraits(object):
    __metaclass__ = Interposer
```

I think